tools/gmhazard_scripts/db_creation/empirical_db/empirical_model_configs/22p4.yaml

```diff
@@ -29,7 +29,6 @@
 ACTIVE_SHALLOW:
   PGV:
     geom:
-      - Br_10  # Br_10 or BSSA_14 or CB_14 or CY_14 or ASK_14
```

Comment:
Can we put this change in a new file. Can we also leave it as a comment?

Reply:
Sure, tt can be done :+1: